### PR TITLE
docs: telemetry reference page and sync service config docs.

### PR DESCRIPTION
**Pull request by thruflo**

### CODE REVIEW

File: docs/reference/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+---
+
+ElectricSQL collects anonymous usage data and sends them to our telemetry service. This page documents the telemetry approach, what data is collected and how to opt-out.
+
+## Why does ElectricSQL collect telemetry?
+
+We collect telemetry to understand key usage metrics. Specifically, the key metrics are:
+
+1. how many applications are using the Electric sync service
+2. what volume of data are they're syncing
```
Comment (alco) [inline docs/reference/telemetry.md:14]:
```suggestion
2. what volume of data are they syncing
```

**Review by samwillis — APPROVED** [review]
Looks good to me

**Review by alco — APPROVED** [review]

File: docs/api/service.md [hunk 2]
```diff
@@ -53,6 +53,17 @@ When `AUTH_MODE=insecure`, all other authentication options can be omitted.
 | `AUTH_JWT_ISS`                           | <p>This optional setting allows you to specificy the "issuer" that will be matched against the `iss` claim extracted from auth tokens.</p><p>This can be used to ensure that only tokens created by the expected party are used to authenticate your client.</p>                                                                                                                  |
 | `AUTH_JWT_AUD`                           | <p>This optional setting allows you to specificy the "audience" that will be matched against the aud claim extracted from auth tokens.</p><p>This can be used to ensure that only tokens for a specific application are used to authenticate your client.</p>                                                                                                                     |
 
+### Telemetry
+
+By default, ElectricSQL collects aggregated, anonymous usage data and sends them to our telemetry service. See <DocPageLink path="reference/telemetry" /> for more information.
```
Comment (balegas) [inline docs/api/service.md:58]: is it 'them' or 'it'?

**Comment by balegas** [comment]
well done.

**Pull request merged**